test/appengine/test_urlfetch.py

```diff
@@ -0,0 +1,51 @@
+from . import MockResponse, activate_sandbox
+activate_sandbox()
```

Comment:
Let me try, the sandbox is really designed to be activated before anything is imported at all, so that could cause issues.